Extension/src/LanguageServer/client.ts

```diff
@@ -336,6 +339,7 @@ class DefaultClient implements Client {
                     // Listen for messages from the language server.
                     this.registerNotifications();
                     this.registerFileWatcher();
+                    console.log('registered stuff');
```

Comment:
probably don't want to keep these logs. :)

Reply:
Oh, I thought those logs were intentional. Although "registered stuff" seemed odd.